components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1154,6 +1159,40 @@ interface AssetRatioService {
   GetCoinMarkets(string vs_asset, uint8 limit) => (bool success, array<CoinMarket> values);
 };
 
+interface MeldIntegrationService {
+  // Obtains the list of Service Providers
+  // All parameters are optional
+  GetServiceProviders(MeldFilter filter)
```

Comment:
When calling this we need to make sure we're rendering the images in chrome-untrusted frames, but that's to be handled in other PRs.